Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm

```diff
@@ -1301,12 +1329,8 @@ - (NSArray *)accessibilityAttributeNames
             objectAttributes = outlineRowAttrs.get().get();
         else
             objectAttributes = tableRowAttrs.get().get();
-    } else if (backingObject->isTreeItem()) {
-        if (backingObject->supportsCheckedState())
-            objectAttributes = [outlineRowAttrs.get() arrayByAddingObject:NSAccessibilityValueAttribute];
```

Comment:
do we not need this one anymore? couldn't tell if it was replaced by something somewhere else